detection_rules/rule.py

```diff
@@ -966,17 +985,32 @@ def get_version_space(self) -> Optional[int]:
 
     @property
     def latest_version(self) -> Optional[int]:
-        """Retrieve the latest known version of the rule."""
-        min_stack = self.get_supported_version()
-        return self.version_lock.get_locked_version(self.id, min_stack)
+        """Retrieve the latest known version of the rule, considering the version lock setting."""
+        toml_version = self.data.get("version")
+
+        if BYPASS_VERSION_LOCK:
+            return toml_version or 1
```

Comment:
We had discussed whether to set to 1 or encourage (enforce) stripping. I saw your comment that `null` is not valid.

This is what I was referring to:

https://github.com/elastic/detection-rules/blob/baf2188a06b4035ecffb7e076168a0520142957e/detection_rules/rule.py#L1275-L1287

we should make the default behavior to strip, like:

```python
include_version: bool = not BYPASS_VERSION_LOCK
```

This would just exclude it by default